src/deprecations.jl

```diff
@@ -8,3 +8,6 @@ import Base: push!
 @deprecate faces(ip::Interpolation) facedof_indices(ip) false
 @deprecate edges(ip::Interpolation) edgedof_indices(ip) false
 @deprecate nfields(dh::AbstractDofHandler) length(getfieldnames(dh)) false
+@deprecate getcoordinates(node::Node) get_node_coordinate(node) false
+@deprecate getcoordinates(args...) get_cell_coordinates(args...) false
+@deprecate getcoordinates!(args...) get_cell_coordinates!(args...) false
```

Comment:
I think the false here has to be removed, so users can see the deprecation.

Reply:
I didn't know this affected visibility, I thought it only caused export or not.
So for `cellcoords!` (now below), it should be false I think? 
(since that wasn't exported before I don't want to export it now)

Reply:
The last argument is whether the old name should be exported or not.